azurerm/resource_arm_servicebus_subscription.go

```diff
@@ -82,6 +82,12 @@ func resourceArmServiceBusSubscription() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"forward_to": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The name of a Queue or Topic to automatically forward messages to.",
```

Comment:
as discussed on Slack, we don't generally specify the description for schema fields in the Azure Provider - we may in the future when there's a value for it (but there's no harm in leaving this in 😄)